packages/renderer/.eslintrc.json

```diff
@@ -1,7 +1,7 @@
 {
   "env": {
     "browser": true,
-    "node": false
+    "node": true
```

Comment:
We do not accept nodejs code on renderer in this new Vite structure

Reply:
@baruchiro fixed

Reply:
OK, it is just one thing I saw in quick review. I will review it... I guess... at the end of the week I hope